tests/lib/test_config.py

```diff
@@ -0,0 +1,8 @@
+# pylint: disable=missing-function-docstring disable=missing-module-docstring
+
+import nestor_api.lib.config as config
+
+
+def test_for_app():
+    configuration = config.for_app("my-app")
+    assert configuration == dict({"app_name": "my-app"})
```

Comment:
I think `dict` is redundant here 🙂  (but since it is a placeholder you don't have to fix it).
```suggestion
def test_for_app():
    configuration = config.for_app("my-app")
    assert configuration == {"app_name": "my-app"}
```